cmd/query/app/apiv3/otlp_translator.go

```diff
@@ -112,7 +112,7 @@ func jSpanToOTLP(jSpan *model.Span) (*tracev1.Span, resource, instrumentationLib
 		Kind:              tracev1.Span_SPAN_KIND_INTERNAL,
 	}
 	if kind, found := jSpan.GetSpanKind(); found {
-		s.Kind = jSpanKindToInternal(kind)
+		s.Kind = jSpanKindToInternal(kind.String())
```

Comment:
isn't this now simply `s.Kind = kind`? And `jSpanKindToInternal()` not neeed?

Reply:
I think it's needed. s.Kind is defined as `tracev1.Span_SpanKind` whereas kind is `trace.SpanKind`. both are different.

Reply:
then at least let's not do it based on the string.